### PR TITLE
[CVP-2489] Increase the wait time for custom operator scorecard tests

### PR DESCRIPTION
**Pull request by dirgim**
* Add the wait-time option to the custom scorecard test and set it to 3000s

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 1]
```diff
@@ -50,5 +50,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
     --namespace="${NAMESPACE}" \
     --kubeconfig "${KUBECONFIG}" \
     --output json \
+    --wait-time 3000s \
```
Comment (lslebodn) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:53]: Please add a comment why such value was used and ideally explanation why it should not be increased

Comment (dirgim) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:55]: Added the explanation above the command

Comment (lslebodn) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:55]: LGTM

**Review by nmars — APPROVED** [review]
lgtm

**Pull request merged**